src/test/java/duke/RichieTest.java

```diff
@@ -0,0 +1,17 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RichieTest {
+
+    @Test
+    void run() {
```

Comment:
The method for tests should be in the format: `featureUnderTest_testScenario_expectedBehavior()`